src/components/tabs/__internal__/tab-title/tab-title.component.tsx

```diff
@@ -258,6 +274,12 @@ const TabTitle = React.forwardRef(
       },
       onFocus: showTooltip,
       onBlur: hideTooltip,
+      ...(validationRedesignOptIn &&
+        hasFailedValidation && {
+          "aria-invalid": true,
+          "aria-errormessage": `${id}-message`,
```

Comment:
**issue:** Looks like [aria-errormessage](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-errormessage#:~:text=When%20we%20went,as%20alert.) may cause axe issues if not used with an announcement method like aria-live, this may be something that needs input from the accessibility team.

Reply:
Decided to use the `aria-describedby` announcement method, since this does not cause any other issues.